e2etests/utils_test.go

```diff
@@ -0,0 +1,25 @@
+// +build e2e slim_e2e
+
+package e2etests
+
+import (
+	"testing"
+
+	v1 "github.com/stackrox/scanner/api/v1"
```

Comment:
We don't need to translate the module's name here.

Reply:
this is done automatically, idk why. it's like this everywhere and this was just copied over from elsewhere, so i'm going to keep it